tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -212,6 +212,13 @@ def test_nan_to_num_nan(self):
     def test_nan_to_num_inf_nan(self):
         self.check_unary_inf_nan('nan_to_num')
 
+    def test_nan_to_num_nan_arg(self):
+        self.check_unary_nan('nan_to_num', kwargs={'nan': 1.0})
```

Comment:
How about
```suggestion
        self.check_unary_nan('nan_to_num', nan=1.0)
```
and
```diff
-def check_unary_nan(self, name, xp, dtype, kwargs={}
+def check_unary_nan(self, name, xp, dtype, **kwargs
```

Reply:
Okay! I thought of it first and also that there are some methods, `check_unary` and `check_binary`, that take a keyword argument `no_bool` for another purpose than passing it to the function to test. But maybe overthinking.

https://github.com/cupy/cupy/blob/c93c05cb7d603a2b30d7b81cf25f31d039d35873/tests/cupy_tests/math_tests/test_misc.py#L17-L21